examples/server_async.py

```diff
@@ -72,7 +72,7 @@ def setup_server(description=None, context=None, cmdline=None):
         # This is because many devices exhibit this kind of behavior (but not all)
         if args.store == "sequential":
             # Continuing, use a sequential block without gaps.
-            datablock = ModbusSequentialDataBlock(0x00, [1, 0 , 0] * 100)
+            datablock = ModbusSequentialDataBlock(0x00, [17] * 100)
```

Comment:
you really got me confused with this change:

v3.5.4 have
´´´
if args.store == "sequential":
            # Continuing, use a sequential block without gaps.
            datablock = ModbusSequentialDataBlock(0x00, [17] * 100)
´´´

current dev have:
´´´
        if args.store == "sequential":
            # Continuing, use a sequential block without gaps.
            datablock = ModbusSequentialDataBlock(0x00, [17] * 100)
´´´

so I am not really sure from where you got the "[1, 0, 0]" ?

Reply:
Yeah that one is a wrong commit that should be squash with the next fixup. I don't think this commit is necessary anymore after retesting everything with the correct version.